utils/crypto/bls/public.go

```diff
@@ -9,7 +9,9 @@ import (
 	blst "github.com/supranational/blst/bindings/go"
 )
 
-const PublicKeyLen = blst.BLST_P1_COMPRESS_BYTES
+const (
+	PublicKeyLen = blst.BLST_P1_COMPRESS_BYTES
+)
```

Comment:
Just to reduce the diff
```suggestion
const PublicKeyLen = blst.BLST_P1_COMPRESS_BYTES
```